packages/new-block-stream/README.md

```diff
@@ -4,6 +4,87 @@ A tool for creating a stream of new block data.
 
 ## Usage
 
+### Getting started
+
+Install via npm:
+
+```js
+npm install @drizzle-utils/new-block-stream
+```
+
+Usage example (taken from `@drizzle-utils/new-block-stream/example.js`)
+
 ```js
-// TODO
+/* eslint-disable no-console */
+const Web3 = require("web3");
+const createNewBlock$ = require("@drizzle-utils/new-block-stream");
+
+const main = async () => {
+  // const web3 = new Web3("http://127.0.0.1:9545"); // HttpProvider
```

Comment:
We can probably remove this line, the `only used if non-WebsocketProvider` comment below is sufficient.